config/src/main/java/io/scalecube/config/ConfigRegistry.java

```diff
@@ -241,6 +241,91 @@ static ConfigRegistry create(ConfigRegistrySettings settings) {
    */
   List<Integer> intListValue(String name, List<Integer> defaultValue);
 
+  /**
+   * Returns dynamic generic-typed multimap property.
+   *
+   * @param name property name
+   * @return property instance
+   */
+  MultimapConfigProperty<String> stringMultimapProperty(String name);
+
+  /**
+   * Returns current value of multimap property or defaults.
+   *
+   * @param name property name
+   * @param defaultValue default property value
+   * @return property value
+   */
+  Map<String, List<String>> stringMultimapValue(String name, Map<String, List<String>> defaultValue);
+
+  /**
+   * Returns dynamic generic-typed multimap property.
+   *
+   * @param name property name
+   * @return property instance
+   */
+  MultimapConfigProperty<Double> doubleMultimapProperty(String name);
+
+  /**
+   * Returns current value of multimap property or defaults.
+   *
+   * @param name property name
+   * @param defaultValue default property value
+   * @return property value
+   */
+  Map<String, List<Double>> doubleMultimapValue(String name, Map<String, List<Double>> defaultValue);
+
+  /**
+   * Returns dynamic generic-typed multimap property.
+   *
+   * @param name property name
+   * @return property instance
+   */
+  MultimapConfigProperty<Long> longMultimapProperty(String name);
+
+  /**
+   * Returns current value of multimap property or defaults.
+   *
+   * @param name property name
+   * @param defaultValue default property value
+   * @return property value
+   */
+  Map<String, List<Long>> longMultimapValue(String name, Map<String, List<Long>> defaultValue);
+
+  /**
+   * Returns dynamic generic-typed multimap property.
+   *
+   * @param name property name
+   * @return property instance
+   */
+  MultimapConfigProperty<Integer> intMultimapProperty(String name);
+
+  /**
+   * Returns current value of multimap property or defaults.
+   *
+   * @param name property name
+   * @param defaultValue default property value
+   * @return property value
+   */
+  Map<String, List<Integer>> intMultimapValue(String name, Map<String, List<Integer>> defaultValue);
+
+  /**
+   * Returns dynamic generic-typed multimap property.
+   *
+   * @param name property name
+   * @return property instance
+   */
+  MultimapConfigProperty<Duration> durationMultimapProperty(String name);
+
+  /**
+   * Returns current value of multimap property or defaults.
+   *
+   * @param name property name
+   * @param defaultValue default property value
+   * @return property value
+   */
+  Map<String, List<Duration>> durationMultimapValue(String name, Map<String, List<Duration>> defaultValue);
+
```

Comment:
Can u give me a favor and add `durationListProperty` and `durationListValue`   since the code which works for it is already at place (see `ListConfigPropertyImpl.getListPropertyClass()`).

Reply:
done